### ISSUE TITLE
Cannot create a runtime: Instantiation("Instantiation: Module __wbindgen_placeholder__ not found")

### ISSUE THREAD

**Issue by Immanuel-john** [paritytech/subport#245]
when I start the node using below command , the node is not starting and throwing the error

`./target/release/vtb-node  --base-path /tmp/vtb-node/tmp01 --chain local  --charlie --port 30335 --ws-port 9947 --rpc-port 9935 --validator --offchain-worker Never`

`Error: Service(Client(Execution(RuntimeConstruction(Instantiation("Instantiation: Module __wbindgen_placeholder__ not found")))))`

**Comment by Immanuel-john** [comment]
> Check your Cargo.lock

`[[package]]
name = "getrandom"
version = "0.2.3"
source = "registry+https://github.com/rust-lang/crates.io-index"
checksum = "7fcd999463524c52659517fe2cea98493cfe485d10565e7b0fb07dbba7ad2753"
dependencies = [
 "cfg-if 1.0.0",
 "js-sys",
 "libc",
 "wasi 0.10.0+wasi-snapshot-preview1",
 "wasm-bindgen",
]`

this is one of the package that is using wasm-bindgen in cargo.lock. likewise there are 22 packages that are using this wasm-bindgen in cargo.lock. what should i do?

**Comment by NukeManDan** [comment]
Are you trying to use tokio _inside_ your runtime?
https://github.com/tokio-rs/tokio/issues/888

I could be wrong, but this likely is not what you should be doing.

**Comment by Immanuel-john** [comment]
> Are you trying to use tokio _inside_ your runtime? [tokio-rs/tokio#888](https://github.com/tokio-rs/tokio/issues/888)
> 
> I could be wrong, but this likely is not what you should be doing.

No Tokio i am not using. I am getting this error after using this below features from sp-io

`sp-io = { version = '3.0', default-features = false, features = ["disable_panic_handler", "disable_oom"]}`

these feature in sp-io i added because i was facing some other issue like below 
``duplicate lang item in crate 'std' (which 'myexternalcrate' depends on): 'panic_impl'`

so when i added "disable_panic_handler" feature that error was solved. 

initially when my node was running fine in sp-io i didn't add any features. There was a requirement where I need to use U256 type for that i imported primitive-types crate. Once i imported primitive-types crate i faced below issue

**Comment by NukeManDan** [comment]
https://stackoverflow.com/questions/59388952/how-to-solve-substrate-duplicate-lang-item-in-crate-std-which-myexternalcra

You likely have either not feature gated your cargo deps to be `no_std` compliant, or are targeting multiple upstream versions of substrate. check your lock file for substrate commits used, and ensure there are no duplicate items like `sp_io` and the primatives, etc.

**Comment by Immanuel-john** [comment]
> https://stackoverflow.com/questions/59388952/how-to-solve-substrate-duplicate-lang-item-in-crate-std-which-myexternalcra
> 
> You likely have either not feature gated your cargo deps to be `no_std` compliant, or are targeting multiple upstream versions of substrate. check your lock file for substrate commits used, and ensure there are no duplicate items like `sp_io` and the primatives, etc.

Initially when my node was running fine, there was a requirement where I need to use U256 
type. For that I imported primitive-types crate. Once i imported primitive-types crate I faced below issue

([](https://users.rust-lang.org/t/serializing-issue-with-primitive-types-u256-in-a-struct/65280?u=immanuel-john)
)

After that issue is solved, I faced below issue :

``duplicate lang item in crate 'std' (which 'myexternalcrate' depends on): 'panic_impl'`

For that I added "disable_panic_handler" feature and that error also was solved. 

So after these above issue,  I am facing this issue, node is not starting up but there is no compilation errors as such above.
